app/views/users/_profile.html.erb

```diff
@@ -37,7 +37,52 @@
   </div>
 
   <% profile = @user.profile %>
+
+  <div class="crayons-card crayons-card--content-rows">
```

Comment:
Apparently, we still used the profile fields to generate the form for these attributes so now that we remove them we have to hardcode this.